### ISSUE TITLE
Cannot get stream manifest of a video that is still being processed

### ISSUE THREAD

**Issue by Mysterium-Mystery** [Tyrrrz/YoutubeExplode#734]
### Version

Newest (6.3.3)

### Platform

Windows 10

### Steps to reproduce

```
var Vid = await YT.Videos.GetAsync(Link);`
var Manifests = await YT.Videos.Streams.GetManifestAsync(Vid.Id);
```

Using "https://www.youtube.com/watch?v=oKbKHd7nd0o" as Link. The second line of code from above will then throw an error

### Details

YoutubeExplode always worked for me without any issues. Even now, it still works with pretty much every video. Only this one doesn't seem to work
I do sort of have a theory why this might be the case though. The stream is still relatively new (It ended about 8 hours ago) and the chat is still not visible so it hasn't been processed by YouTube. So maybe YTExplode just can't get the manifest for streams which are still being processed. Can someone confirm if that's correct?
If it is, of course it would be a nice feature to be able to download unprocessed streams but if that's too hard to do, that's okay but I think this should at least be noted somewhere

The exception:
YoutubeExplode.Exceptions.VideoUnplayableException: Video 'oKbKHd7nd0o' does not contain any playable streams.
   at YoutubeExplode.Videos.Streams.StreamClient.GetManifestAsync(VideoId videoId, CancellationToken cancellationToken) in /_/YoutubeExplode/Videos/Streams/StreamClient.cs:line 246

### Checklist

- [X] I have looked through existing issues to make sure that this bug has not been reported before
- [X] I have provided a descriptive title for this issue
- [X] I have made sure that that this bug is reproducible on the latest version of the package
- [X] I have provided all the information needed to reproduce this bug as efficiently as possible
- [ ] I have sponsored this project

**Comment by Mysterium-Mystery** [comment]
Ok, the download works now so my theory seems to be correct (YouTube reaaaally took it's time with this one). I'll still leave the issue open because the underlying problem still exists

**Comment by Tyrrrz** [comment]
You are likely correct. The challenge is that there's only a very small window where this issue is observed, so it's hard to test against it.

I think, when the video is being processed, it may be possible to fetch it via segmented streams available in DASH, but it's currently not supported, and I'm not sure if that's even going to help.

**Comment by caverna** [comment]
> You are likely correct. The challenge is that there's only a very small window where this issue is observed, so it's hard to test against it.
> 
> I think, when the video is being processed, it may be possible to fetch it via segmented streams available in DASH, but it's currently not supported. And I'm not sure if that's actually going to work at all.

You can test that using any live stream, it gives the same error as reported...

**Comment by caverna** [comment]
Here's the output from a live video at VideoController.cs, at line 93 (`response.Content.ReadAsStringAsync()`):

`{
  "responseContext": {
    "visitorData": "Cgs3Z2w1eUZabDZhWSjk4YqpBjIICgJERRICEgA%3D",
    "maxAgeSeconds": 0
  },
  "playabilityStatus": {
    "status": "OK",
    "playableInEmbed": true
  },
  "streamingData": {
    "expiresInSeconds": "21540",
    "dashManifestUrl": "https://manifest.googlevideo.com/api/manifest/dash/expire/1696793925/ei/5LAiZdCiO_PD6dsPu5mesAg/ip/2003%3Acb%3Ad71b%3Ac477%3A3d7f%3A3d1b%3A7946%3A7b99/id/1U72z0ByXhI.2/source/yt_live_broadcast/requiressl/yes/as/fmp4_audio_clear%2Cwebm_audio_clear%2Cwebm2_audio_clear%2Cfmp4_sd_hd_clear%2Cwebm2_sd_hd_clear/vprv/1/pacing/0/itag_bl/374%2C375%2C376%2C377%2C384%2C385%2C407%2C408%2C409%2C410%2C411%2C412%2C557%2C558%2C612%2C613%2C617%2C619%2C623%2C628%2C655%2C656%2C660%2C662%2C666%2C671/keepalive/yes/fexp/24007246/beids/24350018/itag/0/playlist_type/DVR/sparams/expire%2Cei%2Cip%2Cid%2Csource%2Crequiressl%2Cas%2Cvprv%2Citag%2Cplaylist_type/sig/AGM4YrMwRAIgAlPwNmdf4aQxKCYR1dlbGplxZDiy5pAu8gV9ZcJQ6QwCID7IcMRRYvNFak6-f3yKM3OCA-TkwQd7txM82FjCX4Jl",
    "hlsManifestUrl": "https://manifest.googlevideo.com/api/manifest/hls_variant/expire/1696793925/ei/5LAiZdCiO_PD6dsPu5mesAg/ip/2003%3Acb%3Ad71b%3Ac477%3A3d7f%3A3d1b%3A7946%3A7b99/id/1U72z0ByXhI.2/source/yt_live_broadcast/requiressl/yes/hfr/1/vprv/1/go/1/pacing/0/nvgoi/1/keepalive/yes/fexp/24007246/beids/24350018/dover/11/itag/0/playlist_type/DVR/sparams/expire%2Cei%2Cip%2Cid%2Csource%2Crequiressl%2Chfr%2Cvprv%2Cgo%2Citag%2Cplaylist_type/sig/AGM4YrMwRQIhAIxCNu8dhx0sDOE7k0LsRMWfX_vkdx0fhH-Fr0GOoU_AAiBvSXOuhzA8cn0owKcZs19bhynh0Yi6-DBMK6lcfB_u0A%3D%3D/file/index.m3u8"
  },
  "heartbeatParams": {
    "intervalMilliseconds": "30000",
    "softFailOnError": true,
    "heartbeatServerData": "GAIgAg=="
  },
  "playbackTracking": {
    "videostatsPlaybackUrl": {
      "baseUrl": "https://s.youtube.com/api/stats/playback?cl=570785414&docid=1U72z0ByXhI&ei=5LAiZdCiO_PD6dsPu5mesAg&fexp=v1%2C23946420%2C19788%2C17088%2C14760%2C6588%2C2602%2C26922%2C2780%2C40293%2C3497%2C40082%2C14490%2C4937%2C40927%2C6203%2C54001%2C7577%2C6588%2C2%2C17245%2C12998%2C2876%2C2307%2C71653%2C4956%2C9740%2C278%2C312%2C1225%2C219%2C608%2C9%2C2581%2C472%2C2588%2C3513%2C7%2C1543%2C34388%2C3725%2C22531%2C6998%2C7%2C5%2C510%2C5032%2C4853%2C16697%2C15763%2C1563%2C3532%2C346%2C159%2C124%2C8515%2C1670%2C1359%2C650%2C1045%2C3951%2C2417%2C127%2C130%2C11%2C7%2C9%2C7%2C9%2C12%2C1632%2C9%2C5%2C11%2C66%2C5%2C22%2C3%2C60%2C7%2C10%2C11%2C682%2C17%2C7833%2C5167%2C85%2C825%2C2782%2C15%2C1125%2C738%2C201%2C489%2C2%2C317%2C834%2C505%2C5816%2C3227%2C1088%2C715%2C7%2C22%2C46%2C2540%2C22161%2C64902%2C44033%2C1910%2C261%2C1439%2C14625704%2C786%2C11678694%2C382%2C2158%2C3605%2C119%2C867%2C721%2C75%2C150%2C190%2C388%2C40%2C1760%2C1115%2C1650%2C1170%2C873%2C543%2C184%2C720%2C58%2C166%2C1002%2C1948%2C564%2C1332%2C417%2C375%2C60%2C1603%2C1140%2C1090%2C210%2C46%2C78%2C302%2C498%2C107%2C70%2C1080%2C907%2C1126%2C360%2C1555%2C295%2C611%2C3%2C528&live=dvr&ns=yt&plid=AAYHNJeyUlyJMnx8&delay=5&el=detailpage&len=0&of=qNw2AYySOk82WMEedkE6EA&vm=CAEQARgEOjJBQWpSVTZua1Vwcko2MTd6b2JoZ25XM0JBdHRWOHZ6UzBDdk42UURrNXVjWjFNVmJVQWJyQVBta0tESUFkQnBReV9xb0NaTGg2N09vaWZtemEzXzRFZkxOcURaWEUxazhrQ3ZLdk9uM0lmNjZlNzZleGlBNl8tZl8xdWhWM3c3RXFUZlRjZGpiVXNGdWl1eDdIdDdYang0em1HeER1YWhhSGNRMWt3aAE",
      "headers": [
        {
          "headerType": "USER_AUTH"
        },
        {
          "headerType": "VISITOR_ID"
        }
      ]
    },
    "videostatsDelayplayUrl": {
      "baseUrl": "https://s.youtube.com/api/stats/delayplay?cl=570785414&docid=1U72z0ByXhI&ei=5LAiZdCiO_PD6dsPu5mesAg&fexp=v1%2C23946420%2C19788%2C17088%2C14760%2C6588%2C2602%2C26922%2C2780%2C40293%2C3497%2C40082%2C14490%2C4937%2C40927%2C6203%2C54001%2C7577%2C6588%2C2%2C17245%2C12998%2C2876%2C2307%2C71653%2C4956%2C9740%2C278%2C312%2C1225%2C219%2C608%2C9%2C2581%2C472%2C2588%2C3513%2C7%2C1543%2C34388%2C3725%2C22531%2C6998%2C7%2C5%2C510%2C5032%2C4853%2C16697%2C15763%2C1563%2C3532%2C346%2C159%2C124%2C8515%2C1670%2C1359%2C650%2C1045%2C3951%2C2417%2C127%2C130%2C11%2C7%2C9%2C7%2C9%2C12%2C1632%2C9%2C5%2C11%2C66%2C5%2C22%2C3%2C60%2C7%2C10%2C11%2C682%2C17%2C7833%2C5167%2C85%2C825%2C2782%2C15%2C1125%2C738%2C201%2C489%2C2%2C317%2C834%2C505%2C5816%2C3227%2C1088%2C715%2C7%2C22%2C46%2C2540%2C22161%2C64902%2C44033%2C1910%2C261%2C1439%2C14625704%2C786%2C11678694%2C382%2C2158%2C3605%2C119%2C867%2C721%2C75%2C150%2C190%2C388%2C40%2C1760%2C1115%2C1650%2C1170%2C873%2C543%2C184%2C720%2C58%2C166%2C1002%2C1948%2C564%2C1332%2C417%2C375%2C60%2C1603%2C1140%2C1090%2C210%2C46%2C78%2C302%2C498%2C107%2C70%2C1080%2C907%2C1126%2C360%2C1555%2C295%2C611%2C3%2C528&live=dvr&ns=yt&plid=AAYHNJeyUlyJMnx8&delay=5&el=detailpage&len=0&of=qNw2AYySOk82WMEedkE6EA&vm=CAEQARgEOjJBQWpSVTZua1Vwcko2MTd6b2JoZ25XM0JBdHRWOHZ6UzBDdk42UURrNXVjWjFNVmJVQWJyQVBta0tESUFkQnBReV9xb0NaTGg2N09vaWZtemEzXzRFZkxOcURaWEUxazhrQ3ZLdk9uM0lmNjZlNzZleGlBNl8tZl8xdWhWM3c3RXFUZlRjZGpiVXNGdWl1eDdIdDdYang0em1HeER1YWhhSGNRMWt3aAE",
      "elapsedMediaTimeSeconds": 5,
      "headers": [
        {
          "headerType": "USER_AUTH"
        },
        {
          "headerType": "VISITOR_ID"
        }
      ]
    },
    "videostatsWatchtimeUrl": {
      "baseUrl": "https://s.youtube.com/api/stats/watchtime?cl=570785414&docid=1U72z0ByXhI&ei=5LAiZdCiO_PD6dsPu5mesAg&fexp=v1%2C23946420%2C19788%2C17088%2C14760%2C6588%2C2602%2C26922%2C2780%2C40293%2C3497%2C40082%2C14490%2C4937%2C40927%2C6203%2C54001%2C7577%2C6588%2C2%2C17245%2C12998%2C2876%2C2307%2C71653%2C4956%2C9740%2C278%2C312%2C1225%2C219%2C608%2C9%2C2581%2C472%2C2588%2C3513%2C7%2C1543%2C34388%2C3725%2C22531%2C6998%2C7%2C5%2C510%2C5032%2C4853%2C16697%2C15763%2C1563%2C3532%2C346%2C159%2C124%2C8515%2C1670%2C1359%2C650%2C1045%2C3951%2C2417%2C127%2C130%2C11%2C7%2C9%2C7%2C9%2C12%2C1632%2C9%2C5%2C11%2C66%2C5%2C22%2C3%2C60%2C7%2C10%2C11%2C682%2C17%2C7833%2C5167%2C85%2C825%2C2782%2C15%2C1125%2C738%2C201%2C489%2C2%2C317%2C834%2C505%2C5816%2C3227%2C1088%2C715%2C7%2C22%2C46%2C2540%2C22161%2C64902%2C44033%2C1910%2C261%2C1439%2C14625704%2C786%2C11678694%2C382%2C2158%2C3605%2C119%2C867%2C721%2C75%2C150%2C190%2C388%2C40%2C1760%2C1115%2C1650%2C1170%2C873%2C543%2C184%2C720%2C58%2C166%2C1002%2C1948%2C564%2C1332%2C417%2C375%2C60%2C1603%2C1140%2C1090%2C210%2C46%2C78%2C302%2C498%2C107%2C70%2C1080%2C907%2C1126%2C360%2C1555%2C295%2C611%2C3%2C528&live=dvr&ns=yt&plid=AAYHNJeyUlyJMnx8&el=detailpage&len=0&of=qNw2AYySOk82WMEedkE6EA&vm=CAEQARgEOjJBQWpSVTZua1Vwcko2MTd6b2JoZ25XM0JBdHRWOHZ6UzBDdk42UURrNXVjWjFNVmJVQWJyQVBta0tESUFkQnBReV9xb0NaTGg2N09vaWZtemEzXzRFZkxOcURaWEUxazhrQ3ZLdk9uM0lmNjZlNzZleGlBNl8tZl8xdWhWM3c3RXFUZlRjZGpiVXNGdWl1eDdIdDdYang0em1HeER1YWhhSGNRMWt3aAE",
      "headers": [
        {
          "headerType": "USER_AUTH"
        },
        {
          "headerType": "VISITOR_ID"
        }
      ]
    },
    "ptrackingUrl": {
      "baseUrl": "https://www.youtube.com/ptracking?ei=5LAiZdCiO_PD6dsPu5mesAg&oid=82ZyHtardRAbOneTQSSPtg&plid=AAYHNJeyUlyJMnx8&pltype=contentlive&ptchn=rkRKvn30Y7tzuyOBduzKXA&ptk=youtube_single&video_id=1U72z0ByXhI",
      "headers": [
        {
          "headerType": "USER_AUTH"
        },
        {
          "headerType": "VISITOR_ID"
        }
      ]
    },
    "qoeUrl": {
      "baseUrl": "https://s.youtube.com/api/stats/qoe?cl=570785414&docid=1U72z0ByXhI&ei=5LAiZdCiO_PD6dsPu5mesAg&event=streamingstats&fexp=v1%2C23946420%2C19788%2C17088%2C14760%2C6588%2C2602%2C26922%2C2780%2C40293%2C3497%2C40082%2C14490%2C4937%2C40927%2C6203%2C54001%2C7577%2C6588%2C2%2C17245%2C12998%2C2876%2C2307%2C71653%2C4956%2C9740%2C278%2C312%2C1225%2C219%2C608%2C9%2C2581%2C472%2C2588%2C3513%2C7%2C1543%2C34388%2C3725%2C22531%2C6998%2C7%2C5%2C510%2C5032%2C4853%2C16697%2C15763%2C1563%2C3532%2C346%2C159%2C124%2C8515%2C1670%2C1359%2C650%2C1045%2C3951%2C2417%2C127%2C130%2C11%2C7%2C9%2C7%2C9%2C12%2C1632%2C9%2C5%2C11%2C66%2C5%2C22%2C3%2C60%2C7%2C10%2C11%2C682%2C17%2C7833%2C5167%2C85%2C825%2C2782%2C15%2C1125%2C738%2C201%2C489%2C2%2C317%2C834%2C505%2C5816%2C3227%2C1088%2C715%2C7%2C22%2C46%2C2540%2C22161%2C64902%2C44033%2C1910%2C261%2C1439%2C14625704%2C786%2C11678694%2C382%2C2158%2C3605%2C119%2C867%2C721%2C75%2C150%2C190%2C388%2C40%2C1760%2C1115%2C1650%2C1170%2C873%2C543%2C184%2C720%2C58%2C166%2C1002%2C1948%2C564%2C1332%2C417%2C375%2C60%2C1603%2C1140%2C1090%2C210%2C46%2C78%2C302%2C498%2C107%2C70%2C1080%2C907%2C1126%2C360%2C1555%2C295%2C611%2C3%2C528&live=dvr&ns=yt&plid=AAYHNJeyUlyJMnx8",
      "headers": [
        {
          "headerType": "USER_AUTH"
        },
        {
          "headerType": "VISITOR_ID"
        }
      ]
    },
    "youtubeRemarketingUrl": {
      "baseUrl": "https://www.youtube.com/pagead/viewthroughconversion/962985656/?backend=innertube&cname=30&cver=1_9&foc_id=rkRKvn30Y7tzuyOBduzKXA&label=followon_view&ptype=no_rmkt&random=360925936",
      "elapsedMediaTimeSeconds": 0,
      "headers": [
        {
          "headerType": "USER_AUTH"
        },
        {
          "headerType": "VISITOR_ID"
        }
      ]
    }
  },
  "videoDetails": {
    "videoId": "1U72z0ByXhI",
    "title": "STOCK CAR 2023 AO VIVO | 9ª ETAPA - BUENOS AIRES / ARGENTINA",
    "lengthSeconds": "0",
    "isLive": true,
    "keywords": [
      "stockcar",
      "stock car 2020",
      "Toyota Corolla",
      "Chevrolet Cruze",
      "stock car 2021",
      "calendario 2021",
      "Stock Car ao vivo"
    ],
    "channelId": "UCrkRKvn30Y7tzuyOBduzKXA",
    "isOwnerViewing": false,
    "shortDescription": "#StockCar #StockCar2023\n\nSite Oficial: https://www.stockproseries.com.br/\n\nFacebook: https://www.facebook.com/stockcaroficial/\nInstagram: https://www.instagram.com/stock_car/\nTwitter: https://twitter.com/stock_car\nTikTok: https://www.tiktok.com/@stockcarbrasil",
    "isCrawlable": true,
    "isLiveDvrEnabled": true,
    "thumbnail": {
      "thumbnails": [
        {
          "url": "https://i.ytimg.com/vi/1U72z0ByXhI/default.jpg?v=65206c7c",
          "width": 120,
          "height": 90
        },
        {
          "url": "https://i.ytimg.com/vi/1U72z0ByXhI/mqdefault.jpg?v=65206c7c",
          "width": 320,
          "height": 180
        },
        {
          "url": "https://i.ytimg.com/vi/1U72z0ByXhI/hqdefault.jpg?v=65206c7c",
          "width": 480,
          "height": 360
        },
        {
          "url": "https://i.ytimg.com/vi/1U72z0ByXhI/sddefault.jpg?v=65206c7c",
          "width": 640,
          "height": 480
        }
      ]
    },
    "allowRatings": true,
    "viewCount": "4142",
    "author": "Stock Car",
    "isLowLatencyLiveStream": false,
    "isPrivate": false,
    "isUnpluggedCorpus": false,
    "latencyClass": "MDE_STREAM_OPTIMIZATIONS_RENDERER_LATENCY_NORMAL",
    "isLiveContent": true
  },
  "playerConfig": {
    "audioConfig": {
      "enablePerFormatLoudness": false
    },
    "exoPlayerConfig": {
      "useExoPlayer": true,
      "useAdaptiveBitrate": true,
      "maxInitialByteRate": 91136,
      "minDurationForQualityIncreaseMs": 15000,
      "maxDurationForQualityDecreaseMs": 500,
      "minDurationToRetainAfterDiscardMs": 21000,
      "lowWatermarkMs": 15000,
      "highWatermarkMs": 30000,
      "lowPoolLoad": 0.8,
      "highPoolLoad": 0.8,
      "sufficientBandwidthOverhead": 0.67,
      "bufferChunkSizeKb": 50,
      "httpConnectTimeoutMs": 16000,
      "httpReadTimeoutMs": 8000,
      "numAudioSegmentsPerFetch": 1,
      "numVideoSegmentsPerFetch": 2,
      "minDurationForPlaybackStartMs": 2500,
      "enableExoplayerReuse": false,
      "useRadioTypeForInitialQualitySelection": true,
      "blacklistFormatOnError": false,
      "enableBandaidHttpDataSource": true,
      "httpLoadTimeoutMs": 18000,
      "canPlayHdDrm": true,
      "videoBufferSegmentCount": 389,
      "audioBufferSegmentCount": 38,
      "useAbruptSplicing": false,
      "minRetryCount": 10,
      "minChunksNeededToPreferOffline": 2,
      "secondsToMaxAggressiveness": 0,
      "enableSurfaceviewResizeWorkaround": true,
      "enableVp9IfThresholdsPass": false,
      "matchQualityToViewportOnUnfullscreen": false,
      "lowAudioQualityConnTypes": [
        "MOBILE_CONNECTION_TYPE_CELLULAR_2G",
        "MOBILE_CONNECTION_TYPE_CELLULAR_3G"
      ],
      "useDashForLiveStreams": true,
      "enableLibvpxVideoTrackRenderer": false,
      "lowAudioQualityBandwidthThresholdBps": 0,
      "enableVariableSpeedPlayback": false,
      "preferOnesieBufferedFormat": false,
      "minimumBandwidthSampleBytes": 4096,
      "useDashForOtfAndCompletedLiveStreams": true,
      "disableCacheAwareVideoFormatEvaluation": false,
      "useLiveDvrForDashLiveStreams": true,
      "cronetResetTimeoutOnRedirects": true,
      "emitVideoDecoderChangeEvents": true,
      "onesieVideoBufferLoadTimeoutMs": "0",
      "onesieVideoBufferReadTimeoutMs": "0",
      "libvpxEnableGl": false,
      "enableVp9EncryptedIfThresholdsPass": false,
      "enableOpus": false,
      "usePredictedBuffer": true,
      "maxReadAheadMediaTimeMs": 120000,
      "useMediaTimeCappedLoadControl": true,
      "allowCacheOverrideToLowerQualitiesWithinRange": 0,
      "allowDroppingUndecodedFrames": true,
      "minDurationForPlaybackRestartMs": 5000,
      "serverProvidedBandwidthHeader": "",
      "liveOnlyPegStrategy": "LIVE_ONLY_PEG_STRATEGY_DO_NOT_PEG",
      "enableRedirectorHostFallback": false,
      "enableHighlyAvailableFormatFallbackOnPcr": false,
      "recordTrackRendererTimingEvents": false,
      "minErrorsForRedirectorHostFallback": 0,
      "nonHardwareMediaCodecNames": [
        "OMX.ffmpeg.vp9.decoder",
        "OMX.google.vp9.decoder",
        "c2.android.vp9.decoder",
        "OMX.Intel.sw_vd.vp9",
        "OMX.MTK.VIDEO.DECODER.SW.VP9",
        "c2.mtk.sw.vp9.decoder",
        "OMX.google.av1.decoder",
        "c2.android.av1.decoder",
        "OMX.sprd.av1.decoder"
      ],
      "enableVp9IfInHardware": false,
      "enableVp9EncryptedIfInHardware": false,
      "useOpusMedAsLowQualityAudio": false,
      "minErrorsForPcrFallback": 0,
      "useStickyRedirectHttpDataSource": false,
      "onlyVideoBandwidth": false,
      "useRedirectorOnNetworkChange": false,
      "enableMaxReadaheadAbrThreshold": true,
      "cacheCheckDirectoryWritabilityOnce": false,
      "predictorType": "PREDICTOR_TYPE_NEAREST_RANK",
      "slidingPercentile": 0.5,
      "slidingWindowSize": 12,
      "maxFrameDropIntervalMs": 100,
      "ignoreLoadTimeoutForFallback": false,
      "serverBweMultiplier": 1,
      "drmMaxKeyfetchDelayMs": 0,
      "maxResolutionForWhiteNoise": 0,
      "whiteNoiseRenderEffectMode": "RENDER_EFFECT_MODE_UNKNOWN",
      "enableLibvpxHdr": false,
      "enableCacheAwareStreamSelection": false,
      "useExoCronetDataSource": true,
      "whiteNoiseScale": 0,
      "whiteNoiseOffset": 0,
      "preventVideoFrameLaggingWithLibvpx": false,
      "enableMediaCodecHdr": false,
      "enableMediaCodecSwHdr": false,
      "liveOnlyWindowChunks": 10,
      "bearerMinDurationToRetainAfterDiscardMs": [
        21000,
        21000,
        21000,
        21000,
        120000,
        120000,
        90000,
        120000
      ],
      "forceWidevineL3": false,
      "useAverageBitrate": false,
      "useMedialibAudioTrackRendererForLive": true,
      "useExoPlayerV2": false,
      "logMediaRequestEventsToCsi": false,
      "onesieFixNonZeroStartTimeFormatSelection": false,
      "liveOnlyReadaheadStepSizeChunks": 1,
      "liveOnlyBufferHealthHalfLifeSeconds": 60,
      "liveOnlyMinBufferHealthRatio": 0.75,
      "liveOnlyMinLatencyToSeekRatio": 3,
      "manifestlessPartialChunkStrategy": "ANDROID_PARTIAL_CHUNK_STRATEGY_RETRY_RANGE",
      "ignoreViewportSizeWhenSticky": false,
      "enableLibvpxFallback": false,
      "disableLibvpxLoopFilter": false,
      "enableVpxMediaView": false,
      "hdrMinScreenBrightness": 0,
      "hdrMaxScreenBrightnessThreshold": 0,
      "onesieDataSourceAboveCacheDataSource": false,
      "httpNonplayerLoadTimeoutMs": 72000,
      "numVideoSegmentsPerFetchStrategy": "NUM_SEGMENTS_PER_FETCH_STRATEGY_CONSTANT",
      "maxVideoDurationPerFetchMs": 12000,
      "maxVideoEstimatedLoadDurationMs": -1,
      "estimatedServerClockHalfLife": 5,
      "estimatedServerClockStrictOffset": false,
      "minReadAheadMediaTimeMs": 0,
      "readAheadGrowthRate": 0,
      "useDynamicReadAhead": false,
      "useYtVodMediaSourceForV2": false,
      "enableV2Gapless": false,
      "useLiveHeadTimeMillis": false,
      "allowTrackSelectionWithUpdatedVideoItagsForExoV2": false,
      "maxAllowableTimeBeforeMediaTimeUpdateSec": 0,
      "enableDynamicHdr": false,
      "v2PerformEarlyStreamSelection": false,
      "v2UsePlaybackStreamSelectionResult": false,
      "v2MinTimeBetweenAbrReevaluationMs": 2000,
      "avoidReusePlaybackAcrossLoadvideos": false,
      "enableInfiniteNetworkLoadingRetries": false,
      "reportExoPlayerStateOnTransition": false,
      "manifestlessSequenceMethod": "MANIFESTLESS_SEQUENCE_METHOD_LIVE_HEAD",
      "useLiveHeadWindow": false,
      "enableDynamicHdrInHardware": false,
      "ultralowAudioQualityBandwidthThresholdBps": 0,
      "ignoreUnneededSeeksToLiveHead": false,
      "drmMetricsQoeLoggingFraction": 0.01,
      "useTimeSeriesBufferPrediction": false,
      "slidingPercentileScalar": 1,
      "minAdaptiveVideoQuality": 0,
      "platypusBackBufferDurationMs": 60000
    }
  },
  "trackingParams": "CAAQu2kiEwjQpMe9yeaBAxXzYXoFHbuMB4Y=",
  "adBreakHeartbeatParams": "Q0FBJTNE"
}
`

**Comment by caverna** [comment]
Also if it helps, I noticed that PlayerResponse.cs, line 137, returns 0 streams, but StreamingData has this:
`{
    "expiresInSeconds": "21540",
    "dashManifestUrl": "https://manifest.googlevideo.com/api/manifest/dash/expire/1696797629/ei/Xb8iZfuEGobs1wLlsL_IAw/ip/2003%3Acb%3Ad71b%3Ac477%3A3d7f%3A3d1b%3A7946%3A7b99/id/1U72z0ByXhI.2/source/yt_live_broadcast/requiressl/yes/as/fmp4_audio_clear%2Cwebm_audio_clear%2Cwebm2_audio_clear%2Cfmp4_sd_hd_clear%2Cwebm2_sd_hd_clear/vprv/1/pacing/0/itag_bl/374%2C375%2C376%2C377%2C384%2C385%2C407%2C408%2C409%2C410%2C411%2C412%2C557%2C558%2C612%2C613%2C617%2C619%2C623%2C628%2C655%2C656%2C660%2C662%2C666%2C671/keepalive/yes/fexp/24007246/beids/24350018/itag/0/playlist_type/DVR/sparams/expire%2Cei%2Cip%2Cid%2Csource%2Crequiressl%2Cas%2Cvprv%2Citag%2Cplaylist_type/sig/AGM4YrMwRQIhAIyd-sNaFr4QybfTtK9W4A5KfHBte4MNjuSWXFHREgaAAiA3xU-B63EYxaXnqoRqVh9Oitu-WaRSYHhQBtlg_qAY6w%3D%3D",
    "hlsManifestUrl": "https://manifest.googlevideo.com/api/manifest/hls_variant/expire/1696797629/ei/Xb8iZfuEGobs1wLlsL_IAw/ip/2003%3Acb%3Ad71b%3Ac477%3A3d7f%3A3d1b%3A7946%3A7b99/id/1U72z0ByXhI.2/source/yt_live_broadcast/requiressl/yes/hfr/1/vprv/1/go/1/pacing/0/nvgoi/1/keepalive/yes/fexp/24007246/beids/24350018/dover/11/itag/0/playlist_type/DVR/sparams/expire%2Cei%2Cip%2Cid%2Csource%2Crequiressl%2Chfr%2Cvprv%2Cgo%2Citag%2Cplaylist_type/sig/AGM4YrMwRAIgZ4Qdx-VN-CcepyEaKL7SskbVQ627DtHer_OqDAjB3dQCIGkdVP4DOI-eYExHdng7oMnegA1etxsf_b3rxN4a-XB6/file/index.m3u8"
  }`

**Comment by Tyrrrz** [comment]
> Here's the output from a live video at VideoController.cs, at line 93 (`response.Content.ReadAsStringAsync()`):

Was that for a finished livestream?

**Comment by caverna** [comment]
no, it was during the livestream...

**Comment by Tyrrrz** [comment]
Yeah, an ongoing livestream can only be fetched through the HLS manifest. You can use the `GetHttpLiveStreamUrlAsync(...)` method.

**Comment by Tyrrrz** [comment]
This may have been fixed by #818. Please create a new issue if the problem persists.
